jep/README.adoc

```diff
@@ -1,9 +1,28 @@
 = Index of Jenkins Enhancement Proposals
 
-* link:1/[JEP-1: Jenkins Enhancement Proposal Format]
-* link:2/[JEP-2: Criteria for selecting "Suggested Plugins"]
-* link:200/[JEP-200: Switch Remoting/XStream blacklist to a whitelist]
-* link:201/[JEP-201: Jenkins Configuration as Code]
-* link:3/[JEP-3: Adding the "Released As" field and "Ready for Release" status to Jenkins JIRA]
-* link:300/[JEP-300: Jenkins Essentials]
-* link:301/[JEP-301: Evergreen packaging for Jenkins Essentials]
+[cols="^,>"]
+|===
+| Status | Title
+
+| Active :smile:
+| link:1/[JEP-1: Jenkins Enhancement Proposal Format]
+
+| Draft :speech_balloon:
+| link:2/[JEP-2: Criteria for selecting "Suggested Plugins"]
+
+| Draft :information_source:
```

Comment:
Draft Not Submitted?

Reply:
This file is just auto-generated, I make no claims to the accuracy of any specific JEP 😄

Reply:
Got it, commented on the source at https://github.com/jenkinsci/jep/pull/47/files#r166982163

Reply:
@daniel-beck Why not just _edit_ this PR?  😝 Like so...